test/auro-alert.test.js

```diff
@@ -40,13 +40,13 @@ describe('auro-alert', () => {
     const hiddenContent = root.querySelector('.util_displayHiddenVisually');
 
     await expect(el.role).to.equal("alert");
-    await expect(title.innerHTML).to.equal("Warning");
-    await expect(hiddenContent.innerHTML).to.equal("<!---->Warning.<!---->");
+    await expect(title.innerHTML).to.equal("");
```

Comment:
Please update these tests correctly. Not going to comment on each one individually, but please review the code, the output, and the expected output.

Reply:
Hi @blackfalcon, 
for the line 43, when i check to the code and the file of svg in the alaska-alert.js the title of svg file , the title value is always "" empty string. So can I change into other test?
Thank you

Reply:
You can change the test, but we can't just expect an empty string, because that's not what's happening. The problem is that Lit2.0 went from having `<!------>` to something like `<!--?lit$1565889483$-->` as identifiers in the output. 

So, having a test using `.to.equal` will never work. Try ...

```
await expect(hiddenContent.innerHTML).to.contain("Warning.");
```

Reply:
Hi @blackfalcon, 

for that issue i will fix using the "contain". Thank you for suggest. but I mean for this code : 
await expect(title.innerHTML).to.equal("Warning");
I always get the empty string when i checked the title from SVG file. PFA
<img width="849" alt="title" src="https://github.com/AlaskaAirlines/auro-alert/assets/124669886/538c500a-a830-4595-a4b9-d2677ae4c764">